csrc/rocm/attention.cu

```diff
@@ -316,12 +316,14 @@ __global__ __launch_bounds__(NUM_THREADS) void paged_attention_ll4mi_QKV_kernel(
     int vphysical_blocks[VBLOCKS];
 
     const int warp_start_block_idx = warp_start_token_idx / BLOCK_SIZE;
+    if constexpr (GQA_RATIO < 12) {
```

Comment:
Will not block this merge because of this, but please consider turning this magic 12 into a named constant in a subsequent

Reply:
Noted, thanks!